pkg/ocm/regions/regions.go

```diff
@@ -20,38 +20,27 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/aws/aws-sdk-go/aws/credentials"
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 
 	"github.com/openshift/moactl/pkg/aws"
-	"github.com/openshift/moactl/pkg/logging"
 )
 
 func GetRegions(client *cmv1.Client) (regions []*cmv1.CloudRegion, err error) {
-	// Create logger:
-	logger, err := logging.NewLogger().Build()
-	if err != nil {
-		return nil, fmt.Errorf("Unable to create AWS logger: %v", err)
-	}
-
-	// Create the AWS client:
-	awsClient, err := aws.NewClient().
-		Logger(logger).
-		Region(aws.DefaultRegion).
-		Build()
-	if err != nil {
-		return nil, fmt.Errorf("Failed to create AWS client: %v", err)
-	}
+	// Retrieve AWS credentials from the local AWS user
+	// pass these to OCM to validate what regions are available
+	// in this AWS account
+	localAWSCredentials := credentials.NewEnvCredentials()
```

Comment:
Here we should grab the credentials from the client, which can also honor the `--profile` flag.

Reply:
> Apart from my comment about `GetRegions`, this looks fine to me

Thanks will take a look!